From 12ffd3b6ed2602b5287aaf20071b9a2524c04b7d Mon Sep 17 00:00:00 2001
From: Pavan S <psreenivas@habana.ai>
Date: Thu, 18 Jan 2024 05:53:27 +0200
Subject: [PATCH] [SW-166373] gaudi2: increase tpm init timeout

It is observed that on some host the prstn
deassertion takes more time and accessing tpm
before this causes all transaction to fail.
This patch increases tpm prstn worst case timeout
to 10 sec.

Signed-off-by: Pavan S <psreenivas@habana.ai>
Change-Id: Ifc357a2948c0140477efb1676227d7d038892438
---

diff --git a/arch/gaudi2/common/tpm_auth.c b/arch/gaudi2/common/tpm_auth.c
index 2ef53d0..197d5e8 100644
--- a/arch/gaudi2/common/tpm_auth.c
+++ b/arch/gaudi2/common/tpm_auth.c
@@ -41,7 +41,7 @@
 	return 0;
 }
 
-#define TPM_PRSTN_TIMEOUT_US	1000000	/* 1sec */
+#define TPM_PRSTN_TIMEOUT_US	10000000	/* 10sec */
 int tpm_init(bool print)
 {
 	int ret;
